nexus/test-utils/src/starter.rs

```diff
@@ -560,6 +560,33 @@ impl<'a, N: NexusServer> ControlPlaneStarter<'a, N> {
                 .clone(),
         };
 
+        // Configure the omdb binary path for tests.
```

Comment:
This feels a little sketchy to me - it depends on the changes I made in `config/nextest.toml` - so if there's something more resilient I could / should be doing here, I'm open to it.

Reply:
Yeah that does seem to be a pretty large dependency. That's kind of why we did this -dup thing for reconfigurator-cli, because most tests were purely in memory and we didn't want to have to compile all the db code each iteration.

Reply:
Yeah that does seem to be a pretty large dependency. That's kind of why we did this -dup thing for reconfigurator-cli, because most tests were purely in memory and we didn't want to have to compile all the db code each iteration.